Library/Formula/libspectre.rb

```diff
@@ -0,0 +1,36 @@
+class Libspectre < Formula
+  desc "Small library for rendering Postscript documents"
+  homepage "https://wiki.freedesktop.org/www/Software/libspectre/"
+  url "http://libspectre.freedesktop.org/releases/libspectre-0.2.7.tar.gz"
+  sha256 "e81b822a106beed14cf0fec70f1b890c690c2ffa150fa2eee41dc26518a6c3ec"
+
+  depends_on "ghostscript"
+
+  def install
+    ENV.append "CFLAGS", "-I#{Formula["ghostscript"].opt_include}/ghostscript"
```

Comment:
Unfortunately this appears to be the only way to pass the ghostscript flags. Strictly speaking they are not necessary when the brew prefix is /usr/local, but I added them to take care of users with non-default brew prefixes